src/vue/components/notification-area.vue

```diff
@@ -85,15 +93,18 @@ export default {
 
     showLoginNotification () {
       return Boolean(
-        this.$store.state.auth.notification.visible &&
-        (this.$store.state.auth.notification.count === 1 || this.$store.state.auth.notification.count % 10 === 0)
+        this.$store.state.auth.notification.visible
       )
     }
   },
 
   methods: {
     featureOptionChanged: function (name, value) {
-      this.app.featureOptionChange(name, value)
+      let keyinfo = Options.parseKey(name)
```

Comment:
this is a bug in my prior changes for #522 - wasn't setting the right name here.